### ISSUE TITLE
NEP in CO2 simulation

### ISSUE THREAD

**Issue by ymxu-atm** [geoschem/geos-chem#2385]
### Your name

Yiming

### Your affiliation

Purdue University

### Please provide a clear and concise description of your question or discussion topic.

HI,

I am using GC 14.2.3 to simulate CO2 concentration from 2003 to 2016. In my simulation, I want to change the NEP input with my own data. However, I found that the time range of 'SiB3_3hr_NEP.nc' file is only from 2006 to 2010. I wonder how Geos-chem copes with the NEP input and how I can change it.

Thanks!

**Comment by yantosca** [comment]
Yes, that should work.  You can change the years to suit your data.

**Comment by yantosca** [comment]
Closing out this issue now.
